exe_10.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+
+def sort_axis(array:np, axis1:int)->np:

```

Comment:
Use more specific types for type hinting.
https://numpy.org/devdocs/reference/typing.html